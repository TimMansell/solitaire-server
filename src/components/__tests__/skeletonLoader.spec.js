import { shallowMount } from '@vue/test-utils';
import SkeletonLoader from '@/components/SkeletonLoader.vue';

describe('SkeletonLoader.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(SkeletonLoader);

    expect(wrapper).toMatchSnapshot();
  });
});
