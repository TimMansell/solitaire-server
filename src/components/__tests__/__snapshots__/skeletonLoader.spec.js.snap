// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonLoader.vue matches snapshot 1`] = `<span class="skeleteon"><span class="skeleteon__bar"></span></span>`;
